src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.impl.CsvReaderImpl;
+import core.basesyntax.impl.CsvWriterImpl;
+import core.basesyntax.impl.OperationStrategyImpl;
+import core.basesyntax.impl.ProcessDataImpl;
+import core.basesyntax.impl.ReportCreator;
+import core.basesyntax.impl.ShopUpdateImpl;
+import core.basesyntax.service.CreateReport;
+import core.basesyntax.service.CsvFileReader;
+import core.basesyntax.service.CsvFileWriter;
+import core.basesyntax.service.ProcessData;
+import core.basesyntax.service.ShopUpdateService;
+import core.basesyntax.strategy.Strategy;
+import core.basesyntax.transactor.FruitTransaction;
+import core.basesyntax.transactor.Operation;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        CsvFileReader csvFileReader = new CsvReaderImpl();
+        ProcessData processData = new ProcessDataImpl();
+        Strategy operationStrategy = new OperationStrategyImpl() {
+            @Override
+            public OperationHandler get(Operation operation, Map<Operation, OperationHandler> map) {
+                return null;
+            }
+        };
```

Comment:
Don’t create Strategy variable like an anonymous class. You should pass Strategy map to the constructor.